geometry.pyi

```diff
@@ -194,3 +191,16 @@ class Polygon:
 def regular_polygon(
     sides: int, center: Coordinate, radius: float, angle: float = 0
 ) -> Polygon: ...
+@overload
+def raycast(
+    originpos: Coordinate,
+    endpos: Coordinate,
+    colliders: List[Rect, Circle, Line],
+) -> Coordinate: ...
+@overload
+def raycast(
+    originpos: Coordinate,
+    angle: float,
+    max_dist: float,
+    colliders: List[Rect, Circle, Line],
+) -> Coordinate: ...
```

Comment:
Here you should use Sequence[Rect, Circle, Line]

Reply:
Good catch